### ISSUE TITLE
Ancient Egypt Loot estimate misses last step

### ISSUE THREAD

**Issue by wschaal** [dsiekiera/foe-helfer-extension#1004]
The last task of the Ancient Egypt settlement is to collect 30 of each cultural goods (and some coins). The new Loot calculator does not take this into consideration. The final amount is off by 1200.

I would guess that this was missed for the other settlement types as well.

**Comment by andrgin** [comment]
Yes currently we only show you what is needed to unlock the buildings.
